feature - Allow querying of deleted entities

GtheSheep opened the pull request:
- [ ] Need to test

Closes #2

TimNg-Pair commented:
I changed the `pip_url` in my `meltano.yml` to use `git+https://github.com/GtheSheep/tap-Tiktok.git@feature/include-deleted`, but the numbers still didn't change. I don't think the deleted ads are coming in?

GtheSheep commented:
@TimNg-Pair - I've updated the API Version and switched this branch to use the primary_status, beyond that there's really not much I can get from their API docs so will have to reach out if it doesn't work, if you'd like to try
